include/radix/core/gl/OpenGL.hpp

```diff
@@ -1,6 +1,10 @@
 #ifndef RADIX_CORE_GL_OPENGL_HPP
 #define RADIX_CORE_GL_OPENGL_HPP
 
+#ifdef _WIN32
+#define RADIX_GL_USE_GLAD
```

Comment:
The problem is that the VS project files are **not** generated by CMakeLists (not yet at least). And so for now without a define for glad the VS version won't run anymore. I can maybe change the _WIN32 define to a specific VS define temporarily until we are able to generate the VS solution from CMake.